pkgs/development/tools/rust/cargo-geiger/default.nix

```diff
@@ -1,6 +1,8 @@
 { stdenv, lib, fetchFromGitHub
 , rustPlatform, pkgconfig
-, openssl, Security }:
+, openssl
+# darwin
```

Comment:
Can we remove this if it’s unused?

Reply:
it was just a note to say those are darwin dependencies, did mean the top-lvel import.